smartseq2_single_sample/tasks/picard.wdl

```diff
@@ -4,38 +4,84 @@ task CollectMultipleMetrics {
   String output_filename
   
   command {
-    java -Xmx4g -jar picard.jar CollectMultipleMetrics \
+    java -Xmx6g -jar /usr/picard/picard.jar CollectMultipleMetrics \
       VALIDATION_STRINGENCY=SILENT \
       METRIC_ACCUMULATION_LEVEL=ALL_READS \
-      INPUT=${aligned_bam} \
+      INPUT="${aligned_bam}" \
       OUTPUT="${output_filename}" \
-      FILE_EXTENSION = ".txt" \
-      PROGRAM = CollectAlignmentSummaryMetrics \
-      PROGRAM = CollectInsertSizeMetrics \
-      PROGRAM = CollectGcBiasMetrics \
-      PROGRAM = RnaSeqMetrics \
-      REFERENCE_SEQUENCE=${ref_genome_fasta} \
+      FILE_EXTENSION=".txt" \
+      PROGRAM=null \
+      PROGRAM=CollectAlignmentSummaryMetrics \
+      PROGRAM=CollectInsertSizeMetrics \
+      PROGRAM=CollectGcBiasMetrics \
+      PROGRAM=CollectBaseDistributionByCycle \
+      PROGRAM=QualityScoreDistribution \
+      PROGRAM=MeanQualityByCycle \
+      PROGRAM=CollectSequencingArtifactMetrics \
+      PROGRAM=CollectQualityYieldMetrics \
+      REFERENCE_SEQUENCE="${ref_genome_fasta}" \
       ASSUME_SORTED=true
```

Comment:
I feel like this is not good to include; Shouldn't we assume that the @SO tag contains the correct sort order, and not over-ride it?